code/game/jobs/job/central.dm

```diff
@@ -83,7 +86,8 @@
 	backpack = /obj/item/storage/backpack/satchel
 	backpack_contents = list(
 		/obj/item/clothing/shoes/magboots/advance = 1,
-		/obj/item/storage/box/zipties = 1
+		/obj/item/storage/box/zipties = 1,
+		/obj/item/grenade/bsa = 2
```

Comment:
This might be a prime example of where a subtype might be useful. Specifically a variant that has a configurable blast yield, with configuration working using the same series of prompts that the event 'drop bomb' verb uses.